spec/index.html

```diff
@@ -1031,17 +1026,10 @@ <h3>Fragment identifiers</h3>
        are treated as in RDF syntaxes, as per
        <a data-cite="RDF11-CONCEPTS#section-fragID">RDF 1.1 Concepts and Abstract Syntax</a>
        [[RDF11-CONCEPTS]].
+       and do not follow the process defined for application/yaml.
       </p>
 
-      <p>
-        The YAML media type supports both <a>alias nodes</a> and JSON Pointers [[RFC6905]]
```

Comment:
@gkellogg we don't use the native fragment identifier, so we don't need to explain it.